### PR TITLE
HW03 by Vashkevich Ekaterina

### PR DESCRIPTION
**Pull request by KaterinaVshk**
Implementing the class structure, creating an application, which describes the interaction between students and mentors.

### CODE REVIEW

File: HW03/Ekaterina_Vashkevich/student.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    @name = name
+    @surname = surname
+    @homeworks = []
+    @mentors = []
+    @notifications = []
+  end
+
+  def sumbit_homework!(homework)
+    @homeworks << homework
+    notify_homeworks(homework)
```
Comment (vladhilko) [inline HW03/Ekaterina_Vashkevich/student.rb:16]: What do you think? 🤔 
```suggestion
    notify_mentors(homework)
```

File: HW03/Ekaterina_Vashkevich/student.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  end
+
+  def read_notifications!
+    @notifications.clear
```
Comment (vladhilko) [inline HW03/Ekaterina_Vashkevich/student.rb:20]: maybe?
```suggestion
    notifications.clear
```

File: HW03/Ekaterina_Vashkevich/student.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  def initialize(name, surname)
+    @name = name
+    @surname = surname
+    @homeworks = []
+    @mentors = []
+    @notifications = []
+  end
+
+  def sumbit_homework!(homework)
+    @homeworks << homework
```
Comment (vladhilko) [inline HW03/Ekaterina_Vashkevich/student.rb:15]:
```suggestion
    homeworks << homework
```

File: HW03/Ekaterina_Vashkevich/student.rb (hunk 1 above)
Comment (KaterinaVshk) [reply HW03/Ekaterina_Vashkevich/student.rb:16]: so more clear) thanks)

**Pull request merged**